src/librustdoc/html/static/main.js

```diff
@@ -2627,6 +2623,11 @@ function getSearchElement() {
             option.innerText = crates_text[i];
             elem.appendChild(option);
         }
+
+      var e = getSearchInput();
```

Comment:
At this stage, I think you can used the `search_input` variable created at line 83. Otherwise, seems all good to me, thanks a lot!